hack/minikube.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -e
+set -u
+GIT_COMMIT_ID= git rev-parse --short HEAD
+
+#set namespace here
+kubectl config set-context --current --namespace=default
+
+kubectl create -f deploy/role.yaml
+kubectl create -f deploy/role_binding.yaml
+kubectl create -f deploy/service_account.yaml
+kubectl create -f deploy/crds/operators.coreos.com_servicebindings_crd.yaml
+
+kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sbo-cluster-role
+rules:
+- apiGroups: 
+  - apps.openshift.io
+  resources: 
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - patch
+  - watch
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+EOF
+
+kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sbo-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: service-binding-operator
+  namespace: default #change the namespace if not using the default namespace
+roleRef:
+  kind: ClusterRole
+  name: sbo-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+EOF
+
+sed -i -e 's,REPLACE_IMAGE,"quay.io/redhat-developer/app-binding-operator:$(GIT_COMMIT_ID)",g' ./deploy/operator.yaml
```

Comment:
We should ideally remove the placeholder and have  "quay.io/redhat-developer/app-binding-operator:latest" in the `deploy/operator.yaml` . That breaks our yaml directory otherwise :)

Our scripts might need to adapt to it.